README.md

```diff
@@ -0,0 +1,27 @@
+# Holmes-Analytics
+
+*This project is currently under heavy development and is part of [Google Summer of Code 2017](https://developers.google.com/open-source/gsoc/).*
+
+
+## Overview
+The goal of this project is to implement a semi-generic interface that enables Holmes Processing to manage the execution of advanced statistical and machine learning analysis operations.
```

Comment:
probably link Holmes Processing to the main website: https://holmesprocessing.github.io/